src/cart/crt_group.c

```diff
@@ -3308,15 +3304,14 @@ crt_group_auto_rank_remove(crt_group_t *grp, bool enable)
  * etc...
  */
 int
-crt_group_primary_modify(crt_group_t *grp, crt_context_t *ctxs, int num_ctxs,
-			 d_rank_list_t *ranks, char **uris,
-			 crt_group_mod_op_t op, uint32_t version)
+crt_group_primary_modify(crt_group_t *grp, crt_context_t *ctxs, int num_ctxs, d_rank_list_t *ranks,
+			 uint64_t *incarnations, char **uris, crt_group_mod_op_t op,
+			 uint32_t version)
 {
 	struct crt_grp_priv		*grp_priv;
-	d_rank_list_t			*grp_membs;
 	d_rank_list_t			*to_remove;
-	d_rank_list_t			*to_add;
-	uint32_t			*uri_idx;
+	uint32_t			*to_add;
```

Comment:
probably best to name this as idx_to_add to not confuse with to_remove which is of a different type